CHANGELOG.md

```diff
@@ -2,6 +2,16 @@ postgresql Cookbook CHANGELOG
 =============================
 This file is used to list changes made in each version of the postgresql cookbook.
 
+v4.0.0
+-----
+* No more derived attributes in the attributes file
```

Comment:
I think we need to make it more clear that this is one of the major changes in this version. Maybe something like:

> Potential breaking change: Restructured default attributes to avoid deriving other attribute values from value of the `postgresql.version` attribute.

Are there any related issues that will be closed by this which we can reference here to indicate the driver for this change?